common/source/docs/common-terrain-following.rst

```diff
@@ -85,29 +85,28 @@ the terrain may vary significantly. Key uses are:
    maintain constant height above the ground so you can spend more time
    enjoying the scenary and less time avoiding hills
 
-Sources of terrain data
------------------------
-
-The ground station is responsible for providing the raw terrain data
-which is sent to the aircraft via MAVLink. Right now only MissionPlanner
-(version 1.3.9 or later) and MAVProxy support the required ``TERRAIN_DATA``
-and ``TERRAIN_REQUEST`` messages needed for terrain following support. If
-you are using a different ground station then to load terrain data you
-will need to connect using one of the two support ground stations to
-allow ArduPilot to load terrain data onto your board. It typically takes
-around 2 minutes to load all the terrain data for a mission. Once it is
-loaded it is saved permanently on the microSD card.
-
-Both MissionPlanner and MAVProxy support the global
-`SRTM <https://en.wikipedia.org/wiki/SRTM>`__ database for terrain data.
-That database has a global grid spacing of 3 arc-seconds (around 100
-meters), but has a smaller grid spacing in some parts of the world
-(around 30 meters in the US). Support for other terrain databases can be
-added by extending the ground station code without changes to the
-ArduPilot code.
+
+Sources of Terrain Data
+=======================
+
+The ground station is normally responsible for providing the raw terrain data which is sent to the aircraft via MAVLink. Right now only Mission Planner and MAVProxy support the required TERRAIN_DATA and TERRAIN_REQUEST MAVLink messages needed for terrain following download support. If you are using a different ground station , in order to download terrain data you will need to connect using one of those two ground stations in order to allow ArduPilot to load terrain data onto your board on the ground or in flight.  Once it is loaded, it is saved permanently on the microSD card.
+
+Both MissionPlanner and MAVProxy support the global SRTM database for terrain data. That database has a global grid spacing of 3 arc-seconds (around 100 meters), but has a smaller grid spacing in some parts of the world (around 30 meters in the US).
```

Comment:
our SRTM database only includes the 100m grid. There is a 30m grid available (it is worldwide now I believe), but we don't make it available yet.

Reply:
historical....will update